CDP4Addin/CDP4AddinBootstrapper.cs

```diff
@@ -111,5 +119,28 @@ protected override void ConfigureAggregateCatalog()
 
             logger.Debug("CDP4 Catalogs loaded in: {0} [ms]", sw.ElapsedMilliseconds);
         }
+
+        /// <summary>
+        /// Save the application settings when new plugins are loaded
+        /// </summary>
+        private void SaveSettings(IAppSettingsService<AddinAppSettings> appSettingsService, ICollection<string> newPlugins)
+        {
+            var modules = ServiceLocator.Current.GetAllInstances(typeof(IModule));
```

Comment:
Is it possible to use  
```
ServiceLocator.Current.GetAllInstances<IModule>()
```
Then the type cast ```(IModule)module``` below won't be needed anymore.

Reply:
Again, this variable is an object and cast is needed when is passed to the constructor.

Reply:
@lxatrhea is right